webapp/markdown.py

```diff
@@ -0,0 +1,54 @@
+from mistune import BlockGrammar, BlockLexer, Markdown, _pure_pattern
+import re
+
+
+class DescriptionGrammar(BlockGrammar):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.list_block = re.compile(
+            r"^( *)(?=[*+-•]|\d+\.)(([*+-•])?(?:\d+\.)?) [\s\S]+?"
+            r"(?:"
+            r"\n+(?=\1?(?:[-*_•] *){3,}(?:\n+|$))"  # hrule
+            r"|\n+(?=%s)"  # def links
+            r"|\n+(?=%s)"  # def footnotes\
```

Comment:
These 2 lines seem identical - is this intended?

Reply:
I added a comment to explain where does it comes from :smile:

Reply:
Thanks, that's helpful.